modin/pandas/test/test_groupby.py

```diff
@@ -1111,6 +1119,55 @@ def eval___getattr__(modin_groupby, pandas_groupby, item):
     )
 
 
+def eval__getitem__(md_grp, pd_grp, item):
+    eval_general(
+        md_grp,
+        pd_grp,
+        lambda grp: grp[item].mean(),
+        comparator=build_types_asserter(df_equals),
+    )
+    eval_general(
+        md_grp,
+        pd_grp,
+        lambda grp: grp[item].count(),
+        comparator=build_types_asserter(df_equals),
+    )
+
+    def build_list_agg(fns):
+        def test(grp):
+            res = grp[item].agg(fns)
+            if res.ndim == 2:
+                # Modin's frame has an extra level in the result. Alligning columns to compare.
+                res = res.set_axis(fns, axis=1)
+            return res
+
+        return test
+
+    # issue-#3252
+    eval_general(
+        md_grp,
+        pd_grp,
+        build_list_agg(["mean"]),
+        comparator=build_types_asserter(df_equals),
+    )
+    eval_general(
+        md_grp,
+        pd_grp,
+        build_list_agg(["mean", "count"]),
+        comparator=build_types_asserter(df_equals),
+    )
+    # Explicit default-to-pandas test
```

Comment:
What does this test check?

Reply:
it checks that the actual bug is gone (#3252), initial report was with default-to-pandas reproducer